app/views/hosts/new.html.erb

```diff
@@ -1,7 +1,10 @@
-<% unless (@clone_host && @clone_host.name)
-     title(_("New Host"))
-   else
-     title(_("Clone Host %s") % @clone_host.name)
-   end  %>
+<%
+unless (@clone_host && @clone_host.name)
```

Comment:
Please move view logic to helpers, a good rule of thumb we've been using is to move anything that takes more than 1 line to a helper.